datadog/resource_datadog_monitor.go

```diff
@@ -331,7 +332,15 @@ func resourceDatadogMonitorRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("timeout_h", m.Options.GetTimeoutH())
 	d.Set("escalation_message", m.Options.GetEscalationMessage())
 	d.Set("silenced", m.Options.Silenced)
-	d.Set("include_tags", m.Options.GetIncludeTags())
+	if includeTags, ok := m.Options.GetIncludeTagsOk(); ok {
+		d.Set("include_tags", includeTags)
+	} else {
+		default_, err := res.Schema["include_tags"].DefaultValue()
```

Comment:
This `else` clause is not needed, terraform will do the job of populating the default.